test/CadlRanchProjects/README.md

```diff
@@ -0,0 +1,13 @@
+# How to add a cadl ranch test
+
+1. Checkout this PR by `gh pr checkout 2563`
```

Comment:
Will we still need to checkout the PR when this merges?

Reply:
No. This line will be removed.